uber/tests/models/test_attendee.py

```diff
@@ -104,12 +104,27 @@ def test_is_transferrable(monkeypatch):
     assert Attendee(paid=c.PAID_BY_GROUP).is_transferable
     assert not Attendee(paid=c.NOT_PAID).is_transferable
 
-    assert not Attendee(paid=c.HAS_PAID, trusted=True).is_transferable
     assert not Attendee(paid=c.HAS_PAID, checked_in=datetime.now(UTC)).is_transferable
     assert not Attendee(paid=c.HAS_PAID, badge_type=c.STAFF_BADGE).is_transferable
     assert not Attendee(paid=c.HAS_PAID, badge_type=c.GUEST_BADGE).is_transferable
 
 
+def test_is_not_transferable_trusted(monkeypatch):
+    monkeypatch.setattr(Attendee, 'is_new', False)
+    assert not Attendee(paid=c.HAS_PAID, trusted_depts=c.CONSOLE).is_transferable
+
+
```

Comment:
yea it was originally the way you had it there, and I specifically made it have the word 'error'

I found that a bit easier to read for me personally.  Even knowing what the tests were doing, I thought it was better to be explicit that we were testing for the presence/absence of an error message, and it was a bit more readable.